dvc/analytics/__main__.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+from dvc.analytics import send
+
+
+if __name__ == "__main__":
+    report_fname = sys.argv[1]
```

Comment:
This won't work with binary dvc, as there is no entry point for this separate main. I would prefer keeping it as a part of the daemon, as it was before.

Reply:
@efiop , as far as I understand, adding it as an entry point would be needed if we want to call it as a _script_, like just `analytics`, but I'm calling it as `python -m dvc.analytics`.
Is this an impediment when working with binaries?

Reply:
Maybe i'm missing something, but "python -m dvc.analytics" is calling it as a script. With binary dvc there is no `python` to call from CLI, because it is packed inside of the binary. That is why we had `daemon` subcommand that was simply working through regular `dvc` binary.

Reply:
ohh didn't know that the python command was unavailable.

I agree, having a command for that would "pack" everything under `dvc` )